core/src/main/java/org/apache/accumulo/core/file/blockfile/cache/lru/LruBlockCacheManager.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.accumulo.core.file.blockfile.cache.lru;
+
+import java.util.Map.Entry;
+
+import org.apache.accumulo.core.conf.AccumuloConfiguration;
+import org.apache.accumulo.core.file.blockfile.cache.BlockCache;
+import org.apache.accumulo.core.file.blockfile.cache.BlockCacheManager;
+import org.apache.accumulo.core.file.blockfile.cache.CacheType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class LruBlockCacheManager extends BlockCacheManager {
+
+  private static final Logger LOG = LoggerFactory.getLogger(LruBlockCacheManager.class);
+
+  @Override
+  protected BlockCache createCache(AccumuloConfiguration conf, CacheType type) {
+    LruBlockCacheConfiguration cc = new LruBlockCacheConfiguration(conf, type);
+    LOG.info("Creating {} cache with configuration {}", type, cc);
+    return new LruBlockCache(cc);
+  }
+
+  @Override
+  public void stop() {
+    for (Entry<CacheType,BlockCache> e : this.caches.entrySet()) {
```

Comment:
This code could also be written like the following, and then maybe the map in the superclass does not need to be made visible.

```java
  for(CacheType ct : CacheType.values()) { //not sure if values() is the correct enum method name
      ((LruBlockCache) getBlockCache(ct)).shutdown();
  }
```